atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -281,8 +282,11 @@ private void storeBlock() {
                     }
                     line("} catch (RuntimeException e) {");
                     {
-                        line("log.error(\"Error storing block {} for stream id {}\", row.getBlockId(), row.getId(),"
-                                + " e);");
+                        line("log.error(");
+                        line("        \"Error storing block {} for stream id {}\",");
+                        line("        SafeArg.of(\"blockId\", row.getBlockId()),");
+                        line("        SafeArg.of(\"id\", row.getId()),");
+                        line("        e);");
```

Comment:
The only non-generated changes in this PR are here, where I had to switch to `SafeLogger` to satisfy the error-prone checks. The logs should be backward compatible.